yad: 0.37.0 -> 0.40.0

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools.

This update was made based on information from https://repology.org/metapackage/yad/versions.

These checks were done:

- built on NixOS
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad-icon-browser -h’ got 0 exit code- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad-icon-browser --help’ got 0 exit code
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-icon-browser-wrapped -h’ got 0 exit code
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-icon-browser-wrapped --help’ got 0 exit code- found 0.40.0 with grep in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
- directory tree listing: https://gist.github.com/c49bed23b42e276c3f35734240e17f53

cc @grwlf for review

GrahamcOfBorg commented:
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=4e4e480e-b356-4d14-864f-10f206cb4d02)

The following builds were skipped because they don't evaluate on x86_64-darwin: yad

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=b12b2ced-5077-4bd8-88a9-1801a484822c)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
Wrapping program /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad-icon-browser
Wrapping program /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad
shrinking RPATHs of ELF executables and libraries in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
shrinking /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-wrapped
shrinking /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-icon-browser-wrapped
gzipping man pages under /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/share/man/
strip is /nix/store/fzcs0fn6bb04m82frhlb78nc03ny3w55-binutils-2.28.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin
patching script interpreter paths in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
checking for references to /tmp/nix-build-yad-0.40.0.drv-0 in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0...
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=c70d29ec-9039-4452-96c9-e26728f2e3dc)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
```
</p></details>

GrahamcOfBorg commented:
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=d7a460bb-a10f-4055-8cf2-83d8c75ccbd0)

The following builds were skipped because they don't evaluate on x86_64-darwin: yad

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

Commit:
yad: 0.37.0 -> 0.40.0

Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools.

This update was made based on information from https://repology.org/metapackage/yad/versions.

These checks were done:

- built on NixOS
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad-icon-browser -h’ got 0 exit code
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/yad-icon-browser --help’ got 0 exit code
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-icon-browser-wrapped -h’ got 0 exit code
- ran ‘/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0/bin/.yad-icon-browser-wrapped --help’ got 0 exit code
- found 0.40.0 with grep in /nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
- directory tree listing: https://gist.github.com/0a4215eabbdbe78cf2ad8560c88f7b9c

GrahamcOfBorg commented:
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=fc35b4fa-eee3-41b2-b72f-398d97638baa)

The following builds were skipped because they don't evaluate on x86_64-darwin: yad

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=239ee18b-8b1a-4721-93cf-f3bd280d9801)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/qiw8cbdki0gylr19ryi6204ybp66zni4-yad-0.40.0
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=c513e78f-4aae-4ea0-a9ba-9b744e6ab12e)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling rustc_typeck v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_typeck)
   Compiling rustc_mir v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_mir)
   Compiling rustc_lint v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_lint)
   Compiling rustc_passes v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_passes)
building of '/nix/store/whsyh46wr0zfkw420xcc8ffgahszsd80-rustc-1.24.0.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/nfjkyviyyy9ly6hq1pj5wzy6qxmd491n-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bbqaq71xj30lggbbbf6calm3g7g56c02-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/0hjhsl89xb2yh8m4w00jk2jg1kbqhjzs-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=38729e92-daad-4122-8324-2d5d4e26cfe7)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling rustc_resolve v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_resolve)
   Compiling rustc_passes v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_passes)
   Compiling rustc_lint v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_lint)
   Compiling rustc_mir v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_mir)
building of '/nix/store/whsyh46wr0zfkw420xcc8ffgahszsd80-rustc-1.24.0.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/nfjkyviyyy9ly6hq1pj5wzy6qxmd491n-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bbqaq71xj30lggbbbf6calm3g7g56c02-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/0hjhsl89xb2yh8m4w00jk2jg1kbqhjzs-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38467&attempt_id=fe61a98e-054e-4272-afe8-1fe8766bec0b)

Attempted: yad

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling backtrace v0.3.4
   Compiling proc_macro v0.0.0 (file:///build/rustc-1.24.0-src/src/libproc_macro)
   Compiling rustc_const_math v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_const_math)
   Compiling syntax_ext v0.0.0 (file:///build/rustc-1.24.0-src/src/libsyntax_ext)
building of '/nix/store/whsyh46wr0zfkw420xcc8ffgahszsd80-rustc-1.24.0.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/nfjkyviyyy9ly6hq1pj5wzy6qxmd491n-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bbqaq71xj30lggbbbf6calm3g7g56c02-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/0hjhsl89xb2yh8m4w00jk2jg1kbqhjzs-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/7cprd1vdrs45lczza60zhygh8nlybxfw-yad-0.40.0.drv' failed
```
</p></details>
